lib/mayaUsd/ufe/UsdRotatePivotTranslateUndoableCommand.cpp

```diff
@@ -20,33 +20,44 @@
 MAYAUSD_NS_DEF {
 namespace ufe {
 
-UsdRotatePivotTranslateUndoableCommand::UsdRotatePivotTranslateUndoableCommand(const UsdPrim& prim, const Ufe::Path& ufePath, const Ufe::SceneItem::Ptr& item)
-	: Ufe::TranslateUndoableCommand(item)
-	, fPrim(prim)
-	, fPath(ufePath)
+UsdRotatePivotTranslateUndoableCommand::UsdRotatePivotTranslateUndoableCommand(const Ufe::Path& path)
+	: Ufe::TranslateUndoableCommand(path)
+	, fPath(path)
 	, fNoPivotOp(false)
 {
+	conditionalCreateItem();
+
+
 	// Prim does not have a pivot translate attribute
 	const TfToken xpivot("xformOp:translate:pivot");
-	if (!fPrim.HasAttribute(xpivot))
+	if (!fItem->prim().HasAttribute(xpivot))
 	{
 		fNoPivotOp = true;
 		// Add an empty pivot translate.
-		rotatePivotTranslateOp(fPrim, fPath, 0, 0, 0);
+		rotatePivotTranslateOp(fItem->prim(), fPath, 0, 0, 0);
 	}
 
-	fPivotAttrib = fPrim.GetAttribute(xpivot);
+	fPivotAttrib = fItem->prim().GetAttribute(xpivot);
 	fPivotAttrib.Get<GfVec3f>(&fPrevPivotValue);
 }
 
 UsdRotatePivotTranslateUndoableCommand::~UsdRotatePivotTranslateUndoableCommand()
 {
 }
 
+void UsdRotatePivotTranslateUndoableCommand::conditionalCreateItem()
+{
+    if(!fItem)
+    {
+        auto ufeSceneItemPtr = Ufe::Hierarchy::createItem(fPath);
+        fItem = std::dynamic_pointer_cast<UsdSceneItem>(ufeSceneItemPtr);
+    }
+}
+
```

Comment:
Create on demand: 
On the first access, we create the Ufe::SceneItem from Ufe::Path and then we do a downcast to get the UsdSceneItem and set fItem to it.